app/views/placements/index.html.erb

```diff
@@ -1,40 +1,63 @@
-<h3>Placement Index</h3>
-<br>
-<h4>Click on a placement to view details</h4>
+<div class='mt-4 mb-4'>
+  <h3>Placement Index</h3>
+</div>
 
 
+<h4>Click on a placement to view details</h4>
+
 
+<!-- <div class='row'> -->
   <div class='card bg-light col-6 rounded mb-0'>
-      <div class='card-body'>
+    <div class='card-body'>
         <form action="/" method="get">
           <div class='row form-group'>
             <input type="text" name="search" placeholder="Search" class='form-control col-10 mr-2'>
             <input type="submit" value="Go", class='btn btn-primary col-1'>
           </div>
         </form>
-      </div>
     </div>
+  </div>
   <%= link_to 'New Search', placements_path, class: 'btn btn-primary mt-2' %>
 
 
-
-
 <% if params[:search] %>
-  <div class='card bg-light col-3 rounded mt-2'>
-    <div class='card-body'>
-      <% @placements.each do |p| %>
-        <div>
-          <p><%= link_to p.name, placement_path(p.id) %></p>
-          <%= p.address %>
-          <p><%= p.city %>, <%= p.state %>, <%= p.zip %></p>
-          <%= link_to 'Edit', edit_placement_path(p.id), class: 'btn btn-primary mt-2' %>
+    <% @placements.each do |placement| %>
```

Comment:
I had wanted each result on it's own card, but it may use the space better to put the results all on one cart and make each a li. I'll see how it looks both ways.